sct-commons/src/test/java/org/lfenergy/compas/sct/commons/util/UtilsTest.java

```diff
@@ -482,6 +483,30 @@ void copySclElement_should_throwException() {
                 .hasMessage("org.lfenergy.compas.sct.commons.dto.FCDAInfo is not known to this context");
     }
 
+    @Test
+    void copySclElement_should_succeed_when_syncRead() throws ExecutionException, InterruptedException {
+        // Given
+        TLN tln = new TLN();
+        tln.setLnType("T1");
+        tln.getLnClass().add(TLLN0Enum.LLN_0.value());
+        ExecutorService service = Executors.newFixedThreadPool(2);
+        Callable<TLN> copySclElementTlnCallable = new Callable<TLN>() {
+            @Override
+            public TLN call() {
+                return copySclElement(tln, TLN.class);
+            }
+        };
+        // When
+        List<Future<TLN>> result = service.invokeAll(Arrays.asList(copySclElementTlnCallable, copySclElementTlnCallable));
+        service.shutdown();
```

Comment:
`ExecutorService.shutdown` [javadoc](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/concurrent/ExecutorService.html#shutdown()) :  

> This method does not wait for previously submitted tasks to complete execution. Use [awaitTermination](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/concurrent/ExecutorService.html#awaitTermination(long,java.util.concurrent.TimeUnit)) to do that.

So after shutdown statement, you should add  : 

`assertThat(service.awaitTermination(5, SECONDS)).isTrue();`